backend/src/helpers/new-cars/new-cars-helper.ts.ts

```diff
@@ -0,0 +1,14 @@
+import { getNewCar } from '@services/new-cars/new-cars.service';
+
+import type { ModelResponseDto } from '@autoline/shared/common/types/types';
+
+const newerCars = async (quantity: number): Promise<ModelResponseDto[]> => {
+  const arr = new Array(quantity).fill('');
+  const newCars = await Promise.all(
+    arr.map(async (val, indx) => await getNewCar(-1, indx)),
+  );
+
+  return newCars;
```

Comment:
What is going on here? :eyes: 
Why couldn't you just do a request to DB where you order the cars by the creation date and limit the result to `amount` of the rows?